ydb/public/api/protos/ydb_cms.proto

```diff
@@ -94,6 +94,13 @@ message DatabaseQuotas {
     // A minimum value of `TtlSettings.run_interval_seconds` that can be specified.
     // Default is 1800 (15 minutes).
     uint32 ttl_min_run_internal_seconds = 4;
+    
+    message StoragePoolQuotas {
+        string storage_pool_kind = 1;
+        uint64 data_size_hard_quota = 2;
+        uint64 data_size_soft_quota = 3;
+    }
+    repeated StoragePoolQuotas storage_pools_quotas = 6;
```

Comment:
The name `storage_pools` is misleading, since we don't have storage pools in the public api, we have "storage units". And it's not even about storage pools, but about storage pool kinds. I think it should be simplified to `repeated StorageQuotas storage_quotas` and `unit_kind` instead of `storage_pool_kind` (consistent naming with `StorageUnits`).

Reply:
Changed to the proposed spelling in the newer revision.